README.md

```diff
@@ -1038,6 +1038,7 @@ Manages configuration of a BGP instance.
 |:--------|:-------------|
 | `disable_policy_batching_ipv4` | Not supported on N5k, N6k, N7k |
 | `disable_policy_batching_ipv6` | Not supported on N5k, N6k, N7k |
+| `event_history_periodic      ` | default is 'false' on images I5 and above |
 | `neighbor_down_fib_accelerate` | Not supported on N5k, N6k, N7k |
 | `reconnect_interval`           | Not supported on N5k, N6k, N7k |
 | `suppress_fib_pending`         | Idempotence supported only on I5 images |

```

Comment:
Let's change this to be more explicit: `Idempotence supported only on 7.0(3)I5(1) and later images on n3k and n9k`